Something breaks the Sinon build when using 2.1.0

If updating the referee version from 2.0.0 to 2.1.0 in Sinon, then running the `test-headless` script fails. I am guessing proxyquirify has problems handling the dynamic imports that were introduced.

```
> sinon@5.0.10 test-headless /home/carlerik/dev/sinon
> mochify --recursive -R dot --grep WebWorker --invert --plugin [ proxyquire-universal ] test/**.js

# chromium:
Evaluation failed: Error: Cannot find module './assertions/defined'
      at newRequire (node_modules/proxyquireify/lib/prelude.js:52)
      at req (node_modules/proxyquireify/lib/prelude.js:61)
      at moduleRequire (node_modules/proxyquireify/lib/prelude.js:71)
      at node_modules/@sinonjs/referee/lib/referee.js:239
    at Array.forEach (<anonymous>)
      at Object.require.50../assert-arg-num (node_modules/@sinonjs/referee/lib/referee.js:238)
      at newRequire (node_modules/proxyquireify/lib/prelude.js:75)
      at req (node_modules/proxyquireify/lib/prelude.js:61)
      at moduleRequire (node_modules/proxyquireify/lib/prelude.js:71)
      at Object.<anonymous> (test/assert-test.js:4)
Error: Exit 1
```

--- a/lib/referee.js
+++ b/lib/referee.js
@@ -181,63 +181,59 @@ referee.prepareMessage = function msg(message) {
     return message + (/[.:!?]$/.test(message) ? " " : ": ");
 };
 
-[
-    "./assertions/defined",
-    "./assertions/class-name",
-    "./assertions/contains",
-    "./assertions/equals",
-    "./assertions/exception",
-    "./assertions/greater",
-    "./assertions/has-prototype",
-    "./assertions/is-array",
-    "./assertions/is-array-buffer",
-    "./assertions/is-array-like",
-    "./assertions/is-boolean",
-    "./assertions/is-data-view",
-    "./assertions/is-error",
-    "./assertions/is-eval-error",
-    "./assertions/is-false",
-    "./assertions/is-float-32-array",
-    "./assertions/is-float-64-array",
-    "./assertions/is-function",
-    "./assertions/is-infinity",
-    "./assertions/is-int-8-array",
-    "./assertions/is-int-16-array",
-    "./assertions/is-int-32-array",
-    "./assertions/is-intl-collator",
-    "./assertions/is-intl-date-time-format",
-    "./assertions/is-intl-number-format",
-    "./assertions/is-map",
-    "./assertions/is-nan",
-    "./assertions/is-null",
-    "./assertions/is-number",
-    "./assertions/is-object",
-    "./assertions/is-promise",
-    "./assertions/is-range-error",
-    "./assertions/is-reference-error",
-    "./assertions/is-reg-exp",
-    "./assertions/is-set",
-    "./assertions/is-string",
-    "./assertions/is-symbol",
-    "./assertions/is-syntax-error",
-    "./assertions/is-true",
-    "./assertions/is-type-error",
-    "./assertions/is-uri-error",
-    "./assertions/is-u-int-16-array",
-    "./assertions/is-u-int-32-array",
-    "./assertions/is-u-int-8-array",
-    "./assertions/is-u-int-8-clamped-array",
-    "./assertions/is-weak-map",
-    "./assertions/is-weak-set",
-    "./assertions/keys",
-    "./assertions/less",
-    "./assertions/match",
-    "./assertions/near",
-    "./assertions/same",
-    "./assertions/tag-name"
-].forEach(function (config) {
-    require(config)(referee);
-});
+require("./assertions/defined")(referee);
+require("./assertions/class-name")(referee);
+require("./assertions/contains")(referee);
+require("./assertions/equals")(referee);
+require("./assertions/exception")(referee);
+require("./assertions/greater")(referee);
+require("./assertions/has-prototype")(referee);
+require("./assertions/is-array")(referee);
+require("./assertions/is-array-buffer")(referee);
+require("./assertions/is-array-like")(referee);
+require("./assertions/is-boolean")(referee);
+require("./assertions/is-data-view")(referee);
+require("./assertions/is-error")(referee);
+require("./assertions/is-eval-error")(referee);
+require("./assertions/is-false")(referee);
+require("./assertions/is-float-32-array")(referee);
+require("./assertions/is-float-64-array")(referee);
+require("./assertions/is-function")(referee);
+require("./assertions/is-infinity")(referee);
+require("./assertions/is-int-8-array")(referee);
+require("./assertions/is-int-16-array")(referee);
+require("./assertions/is-int-32-array")(referee);
+require("./assertions/is-intl-collator")(referee);
+require("./assertions/is-intl-date-time-format")(referee);
+require("./assertions/is-intl-number-format")(referee);
+require("./assertions/is-map")(referee);
+require("./assertions/is-nan")(referee);
+require("./assertions/is-null")(referee);
+require("./assertions/is-number")(referee);
+require("./assertions/is-object")(referee);
+require("./assertions/is-promise")(referee);
+require("./assertions/is-range-error")(referee);
+require("./assertions/is-reference-error")(referee);
+require("./assertions/is-reg-exp")(referee);
+require("./assertions/is-set")(referee);
+require("./assertions/is-string")(referee);
+require("./assertions/is-symbol")(referee);
+require("./assertions/is-syntax-error")(referee);
+require("./assertions/is-true")(referee);
+require("./assertions/is-type-error")(referee);
+require("./assertions/is-uri-error")(referee);
+require("./assertions/is-u-int-16-array")(referee);
+require("./assertions/is-u-int-32-array")(referee);
+require("./assertions/is-u-int-8-array")(referee);
+require("./assertions/is-u-int-8-clamped-array")(referee);
+require("./assertions/is-weak-map")(referee);
+require("./assertions/is-weak-set")(referee);
+require("./assertions/keys")(referee);
+require("./assertions/less")(referee);
+require("./assertions/match")(referee);
+require("./assertions/near")(referee);
+require("./assertions/same")(referee);
+require("./assertions/tag-name")(referee);
 
 referee.expect = function () {
     expect.init(referee);

